test/util/IndexTestHelpers.cpp

```diff
@@ -337,7 +340,8 @@ QueryExecutionContext* getQec(TestIndexConfig c) {
                    }},
                    std::make_unique<Index>(makeTestIndex(testIndexBasename, c)),
                    std::make_unique<QueryResultCache>(),
-                   std::make_unique<NamedResultCache>()});
+                   std::make_unique<NamedResultCache>(),
+                   std::make_unique<MaterializedViewsManager>()});
```

Comment:
Same here, at some point we should probably store instances of `Libqlever` here...